src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -139,43 +131,30 @@ export const WorkspaceMenu = ({ basePath, getUrlForApp, workspaces }: Props) =>
   };
 
   const currentWorkspaceButton = (
-    <EuiCollapsibleNavGroup>
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiIcon type="logoOpenSearch" size="l" />
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiText>
-            <strong> {currentWorkspaceName} </strong>
-          </EuiText>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiIcon type="arrowDown" onClick={onButtonClick} />
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    </EuiCollapsibleNavGroup>
-  );
-
-  const currentWorkspaceTitle = (
-    <EuiFlexGroup>
-      <EuiFlexItem>
-        <EuiIcon type="logoOpenSearch" size="l" />
-      </EuiFlexItem>
-      <EuiFlexItem>
-        <EuiText>
-          <strong> {currentWorkspaceName} </strong>
-        </EuiText>
-      </EuiFlexItem>
-      <EuiFlexItem>
-        <EuiIcon type="cross" onClick={closePopover} />
-      </EuiFlexItem>
-    </EuiFlexGroup>
+    <EuiCollapsibleNavGroup
+      title={
+        <h3
+          style={{
+            width: '202px',
```

Comment:
Is is okay to use hardcode width?

Reply:
You're write, I cleanup the custom styles.